applications/PfemMeltingApplication/custom_utilities/heat_source.h

```diff
@@ -85,7 +85,7 @@ namespace Kratos
 
             //node_it->FastGetSolutionStepValue(ARRHENIUS_VALUE)=aux_var_polymer;
 
-            node_it->FastGetSolutionStepValue(HEAT_FLUX) = (-1.0) * density * heat_of_vaporization * aux_var_polymer;
+            node_it->FastGetSolutionStepValue(HEAT_FLUX) = 0.0 * (-1.0) * density * heat_of_vaporization * aux_var_polymer;
```

Comment:
is this temporary? Maybe we can put just
`node_it->FastGetSolutionStepValue(HEAT_FLUX) = 0.0;`